package.json

```diff
@@ -14,6 +14,7 @@
     "build": "yarn generate:definitions && tsc",
     "postversion": "git push && git push --tags",
     "test": "jest -i",
+    "test:integration": "yarn build && cd ./__integrations__ && yarn install --frozen-lockfile && yarn start",
```

Comment:
I'd keep add a `test:all` script that performs unit and integration tests.

Reply:
what about I add a script 
```json
"test:all": yarn test && yarn test:integration
```
?

Reply:
Yes 👍